packages/@okta/vuepress-site/docs/guides/migrate-to-okta/prerequisites/index.md

```diff
@@ -0,0 +1,157 @@
+* Define all user attributes that will be migrated to Okta, and the source of truth for each one.
+
+**Note:** The source of truth for attributes at this stage of the migration process means only that we want to clearly define where the user data is coming from for the migration itself, not where it will be managed after the migration. After migration, Okta is the default [profile master](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-masters.htm), but this can be modified.
+
+### User applications
+
+Identify the applications that each user type can access. Here are some questions to answer:
+
+* What exactly are those applications, and how are they currently accessed?
+* Are users given access to applications based on types or groups, or other user attributes? If so, the use of Okta groups can help with provisioning or application assignment.
+* Do integrations exist on the [OIN (Okta Integration Network)](https://www.okta.com/integrations) for those applications?
+
+### Create a migration plan
+
+After you’ve gathered the necessary information, it’s time to create a migration plan with clear goals in mind. These will provide direction for your plan, as well as offering a way to measure success. Ask yourself these questions:
+
+* How do you want your users to experience the migration?
+* How much and what kind of user impact is acceptable?
+* Will you archive your legacy data?
+* Are there any company policies or standards that you have to take into consideration?
+* Is it acceptable to perform a “one-shot” migration? Using this strategy means short-term downtime, but it could lessen the complexity of the plan.
+* Is it preferable to perform the migration in stages? Depending on the method you use, it could increase complexity of the plan, but service interruptions would be minimized.
+
+### Define attribute mappings
+
+Defining where to map each attribute from the source to the target is an important part of your plan. [Okta user profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-editor.htm#Profiles) include 31 predefined attributes, and more custom attributes can be added as needed.
+
+Using the information you gathered earlier, create a matrix that maps your source attributes to the Okta user profile attributes, and be sure to include these fields:
+
+* Source field
+* Target field
+* Data type
+* Data length
+* Does this field contain PII or other sensitive data?
+* Is this a required field?
+* Is this a unique field?
+* Does the data need transformation of some kind?
+* Is mapping required? (Can it be ignored during migration?)
+
+### Validate data quality
+
+Extract the user data from the source (or sources) into an intermediate staging area. Then you’ll need to clean up that data so that it’s consistent and contains only valid information. Careful review of the data in each field will also help avoid errors when migrating to Okta.
```

Comment:
```suggestion
Extract the user data from the source(s) into an intermediate staging area. Then you’ll need to clean up that data so that it’s consistent and contains only valid information. Careful review of the data in each field will also help avoid errors when migrating to Okta.
```